core/message-handling_test.go

```diff
@@ -388,35 +411,18 @@ func TestMakeApplicableReplicaMessageProcessor(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	processMessage := func(msg interface{}) (new bool, err error) {
-		args := mock.MethodCalled("messageProcessor", msg)
-		return args.Bool(0), args.Error(1)
-	}
 	applyReplicaMessage := func(msg messages.ReplicaMessage) error {
 		args := mock.MethodCalled("replicaMessageApplier", msg)
 		return args.Error(0)
 	}
-	process := makeApplicableReplicaMessageProcessor(processMessage, applyReplicaMessage)
+	process := makeApplicableReplicaMessageProcessor(applyReplicaMessage)
 
 	msg := mock_messages.NewMockReplicaMessage(ctrl)
-	embeddedMsgs := []interface{}{
```

Comment:
By moving this part to `TestMakeReplicaMessageProcessor()`, an element of `embeddedMsgs` has been changed to an empty struct. It is my understanding that there is no one who see the content so there are no difference in what is tested by the codes.

Reply:
You are right, `testify/mock` uses `reflect.DeepEqual` to compare expected arguments which will consider empty structures as equal. Thanks for spotting this!